core/commands.py

```diff
@@ -46,6 +46,10 @@
 ['none']},
 
 					},
+				'osx_x86' :  #generate sub command - os name
+					{
+						'exec' : {'file_to_execute':['none','xor_random','xor_yourvalue','add_random','add_yourvalue','sub_random','sub_yourvalue','inc','inc_timesyouwant','dec','dec_timesyouwant','mix_all']}, #function of shellcode
```

Comment:
Please remove `'xor_random','xor_yourvalue','add_random','add_yourvalue','sub_random','sub_yourvalue','inc','inc_timesyouwant','dec','dec_timesyouwant','mix_all'` encodes and just leave the `none`.